registry.json

```diff
@@ -4365,7 +4366,7 @@
     "coinId": 10000060,
     "symbol": "INJ",
     "decimals": 18,
-    "blockchain": "Cosmos",
+    "blockchain": "NativeInjective",
```

Comment:
why do we need those change(s)?

Reply:
Answered here https://github.com/trustwallet/wallet-core/pull/3498#discussion_r1395713957